exchanges/exchange_test.go

```diff
@@ -2710,13 +2710,18 @@ func TestGetKlineRequest(t *testing.T) {
 	b.Features.Enabled.Kline.ResultLimit = 1439
 	start := time.Date(2020, 12, 1, 0, 0, 0, 0, time.UTC)
 	end := start.AddDate(0, 0, 1)
-	_, err = b.GetKlineRequest(pair, asset.Spot, kline.OneMin, start, end)
+	_, err = b.GetKlineRequest(pair, asset.Spot, kline.OneMin, start, end, true)
```

Comment:
Just for some extra gloriousPettiness, let's include a valid scenario with `true`